odoo/addons/base/tests/test_expression.py

```diff
@@ -1107,14 +1107,17 @@ def test_logic(self):
         with self.assertQueries(['''
             SELECT "res_partner"."id"
             FROM "res_partner"
-            WHERE (("res_partner"."active" = %s) AND (
-                ("res_partner"."name"::text LIKE %s) AND (
+            WHERE (
+                (
+                    ("res_partner"."active" = %s) AND
+                    ("res_partner"."name"::text LIKE %s)
+                ) AND (
                     ("res_partner"."title" = %s) OR (
                         ("res_partner"."ref" != %s) OR
                         "res_partner"."ref" IS NULL
                     )
                 )
-            ))
+            )
```

Comment:
It's still the same query, see this truth-table: https://gist.github.com/Julien00859/856390d6a860742240d8e41875bfcfdc